node/cmd/guardiand/adminclient.go

```diff
@@ -164,10 +164,10 @@ var ClientChainGovernorReleasePendingVAACmd = &cobra.Command{
 }
 
 var ClientChainGovernorResetReleaseTimerCmd = &cobra.Command{
-	Use:   "governor-reset-release-timer [VAA_ID]",
-	Short: "Resets the release timer for a chain governor pending VAA, extending it to the configured maximum",
+	Use:   "governor-reset-release-timer [VAA_ID] <num_days>",
+	Short: "Resets the release timer for a chain governor pending VAA, extending it to num_days (up to a maximum of 7), or 24 hours if num_days is omitted",
```

Comment:
```suggestion
	Short: "Resets the release timer for a chain governor pending VAA, extending it to num_days (up to a maximum of 7), defaulting to one day if num_days is omitted",
```

Switching units between days and hours when the argument is `num_days` is quite confusing.